src/pages/Acknowledgements/pageContent.tsx

```diff
@@ -11,14 +11,14 @@ const BoldCaps = styled.strong`
 `;
 
 export interface PageContentInterface {
-  title: string;
+  title: JSX.Element;
   content: JSX.Element;
-  acknowledgementText?: any;
+  acknowledgementText?: JSX.Element;
 }
 
 export const pageContent = {
   [AcknowledgementIdsEnum.introSection]: {
-    title: 'Proof of stake',
+    title: <FormattedMessage defaultMessage="Proof of stake" />,
```

Comment:
@wackerow why not use the `formatMessage` function here so that the values could remain strings? Was it not possible to use the `useIntl()` hook?

Reply:
@samajammin Exactly. Hooks need to be inside React components... `pageContent` is just an object.